lib/libc/termios/lib_loopback.c

```diff
@@ -0,0 +1,53 @@
+/****************************************************************************
+ *
+ * libc/termios/lib_loopback.c
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <sys/ioctl.h>
+#include <termios.h>
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: loopback
+ *
+ * Description:
+ *   calls the ioctl to enable/ disable the loopback mode of UART
+ *
+ * Input Parameters:
+ *   fd  - The 'fd' argument is an open file descriptor associated with a terminal.
+ *   cmd - ioctl command used to control uart loopback mode.
+ *   termiosp - The termiosp argument is a pointer to a termios structure.
+ *
+ * Returned Value:
+ *   Upon successful completion, 0 is returned. Otherwise, -1 is returned and
+ *   errno is set to indicate the error.
+ *
+ ****************************************************************************/
+
+int loopback(int fd, int cmd, FAR struct termios *termiosp)
```

Comment:
loopback is too common. Many many module can use loopback. 
How about adding `tc` as prefix like other termios functions?

Reply:
I agree with chang's opinion. loopback is too common and ambiguous.
Please add prefix as chang's opinion or verb which indicates what it does

Reply:
corrected